acceptancetests/assess_deploy_webscale.py

```diff
@@ -250,7 +258,7 @@ def parse_args(argv):
 
 
 def mongo_snap_settings(args):
-    if not args.db_snap_path:
+    if not args.db_snap_path and not args.db_asserts_path:
```

Comment:
should we error if you get one and not the other?

Reply:
I think no. I have rigged the CI jobs so that an empty --db-snap-path gets passed when running against the built-in mongo (see https://jenkins.juju.canonical.com/job/MongoWebscaleTests/)

Reply:
Ok, fair enough. Maybe worth a comment in code then to make clear here?

Reply:
Will do!

Reply:
@mitechie Now that I think about it, if you don't provide both, then `juju bootstrap` will fail with an error when validating the arguments so I think we can actually get away without adding a comment here. What do you think?